pkg/collector/corechecks/containers/kubelet/provider/pod/provider_test.go

```diff
@@ -168,11 +168,6 @@ func (suite *ProviderTestSuite) TestTransformRunningPods() {
 	suite.mockSender.AssertNumberOfCalls(suite.T(), "Gauge", 30)
 
 	// 1) pod running metrics
-	suite.mockSender.AssertMetric(suite.T(), "Gauge", common.KubeletMetricsPrefix+"containers.running", 2, "", append(config.Tags, "kube_container_name:prometheus-to-sd-exporter", "kube_deployment:fluentd-gcp-v2.0.10"))
```

Comment:
Instead of removing these, can we replace them with an `AssertMetricNotTaggedWith`?

Reply:
And maybe add a comment before explaining that these containers do not have the namespace tag, so we do not expect these metrics to be emitted?

Reply:
these were accidentally deleted, added them back in! the new test here asserts for metrics that should not be sent:
https://github.com/DataDog/datadog-agent/blob/7b6b49da4c180e08967100da3707239da94bed88/pkg/collector/corechecks/containers/kubelet/provider/pod/provider_test.go#L274-L285